src/main/conversions/v2.23.0/c2_23_0_2018060101.clj

```diff
@@ -0,0 +1,18 @@
+(ns facepalm.c2-23-0-2018060101
```

Comment:
To put it another way: I think this conversion should be `v2.22.0`, unless there's a reason this PR shouldn't be merged until the release after next...